src/App.css

```
@@ -1,38 +1,55 @@
-.App {
-  text-align: center;
-}
+/*gnb 레이아웃: 레이아웃은 앱(부모)의 Gnb 컴포넌트에서 정의*/ 
+.gnb_ly {
+  display: flex;
+  padding: 18px 32px 17px 32px;
+  position: sticky;
+  top: 0;
+  flex-direction: column;
+  align-items: center;
+  align-self: stretch;
+  background: var(--Linkbrary-bg, #F0F6FF);
```

css에서 변수를 선언해놓고 사용할 것이 아니라면 var로 사용하지 않고 그냥 컬러값을 넣어주는게 더 괜찮을 것 같아요.
```suggestion
  background: #F0F6FF;
```